css/framework.css

```diff
@@ -0,0 +1,226 @@
+/* reset */
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+margin: 0;
+padding: 0;
+border: 0;
+font-size: 100%;
+font: inherit;
+vertical-align: baseline;
+}
+.body{
+    font-family: Helvetica Neue,Helvetica,Arial,Utkal,sans-serif;
+}
+/* Row */
+.row{
+    clear: both;
+    width: 100%;
+    height: auto;
+}
+/* Fixed Nav */
+.fixed-nav{
+    clear: both;
+    position: fixed;
+    width: 100%;
+    height: auto;
+    background-color: black;
+    display: flex;
+
+}
+.fixed-nav ul{
```

Comment:
Avoid `.fixed-nav ul` in the framework class
Instead .fixed-nav` in the framework class dont call elements directly.